### DISCUSSION TITLE
milvus 已经移除对 faiss 的依赖了吗

### DISCUSSION THREAD

**Discussion by ponponon** [milvus-io/milvus#26799] [Q&A and General discussion]
现在 milvus 还是使用 faiss 做向量搜索吗？还是自己重写了

**Comment by xiaofan-luan** [comment]
我们依然使用faiss，支持了IVF系列的索引和FastScan索引。

**Reply by xiaofan-luan** [reply]
faiss 的依赖在
https://github.com/zilliztech/knowhere

Milvus会依赖knowhere

**Reply by Escapethebug** [reply]
请问为什么knowhere提供的单元测试gpu比cpu慢，是我测试的不对么还是哪里有问题。

**Reply by xiaofan-luan** [reply]
GPU索引确实不能保证比cpu快。尤其是图索引。GPU索引只在NQ大的情况下有优势。
我建议你根据自己的应用场景建一个benchmark比较下

**Reply by Escapethebug** [reply]
明白！太感谢了！
